rendering_resource_manager_service/service/settings.py

```diff
@@ -45,7 +45,7 @@
 API_VERSION = 'v1'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
Yes, static files are only served when DEBUG is activated. The documentation specifies that Django should NOT be used for production anyway.